_data/alumni.yml

```diff
@@ -476,3 +476,31 @@
     - url: https://www.linkedin.com/in/mehmet-türkmen-324a0838/
       icon: Linkedin
       name: Linkedin
+
+- name: Gwendolyn Chee
+  pic: /img/people/cheeg.jpg
+  dates: 2017-2022
+  role: Graduate Research Assistant
+  description: 'Gwendolyn Chee recieved her MS in Nuclear Engineering in 2018, and graduated from NPRE with her PhD in 2022.'
+  social:
+    - url: mailto:gchee2@illinois.edu
+      icon: envelope-o
+      name: Email
+    - url: http://github.com/gwenchee
+      icon: github-alt
+      name: Github
+
+- name: Gabriele Paciocco
+  pic: /img/people/pacioccog.png
+  dates: 2021-2022
+  role: Undergraduate Research Assistant 
+  description: 'Gabriele is an NPRE undergraduate at UIUC. His interests in Math and Science motivate him to study the future of nuclear reactor devlopement.'
+  social:
+    - url: mailto:Agp9@illinois.edu
+      icon: envelope-o
+      name: Email
+    - url: https://www.linkedin.com/in/gabriele-paciocco-b311b41a1/
+      icon: linkedin
```

Comment:
Is this case-sensitive? The `icon:` on l477 has it capitalized?

Reply:
It looks like he got it from my profile which doesn't have a capitalized L, but other do and it renders on the site so I think we're good either way